llvm/lib/Target/PowerPC/PPCISelDAGToDAG.cpp

```diff
@@ -5475,7 +5475,8 @@ void PPCDAGToDAGISel::Select(SDNode *N) {
     // generate secure plt code for TLS symbols.
     getGlobalBaseReg();
   } break;
-  case PPCISD::CALL: {
+  case PPCISD::CALL:
+  case PPCISD::CALL_RM: {
```

Comment:
Are your targeting this for SPE? SPE should have no floating point instructions? So round mode should not be a concern? For SPE, we may stop generate CALL_RM where it is generated. Note that CALL_RM performs worse than normal CALL even after instruction selection. They are selected to different instructions.

If for other PPC32 targets, the fix looks reasonable to me.

Reply:
> Are your targeting this for SPE?

This is not for SPE.